src/doc/src/reference/rust-version.md

```diff
@@ -0,0 +1,155 @@
+# Rust Version
+
+The `rust-version` field is an optional key that tells cargo what version of the
```

Comment:
Found this :)

```suggestion
The `rust-version` field is an optional key that tells Cargo what version of the
```

Reply:
That was mirroed from the manifest page which uses that a lot.  I'm unsure if I want to go down that rabbit hole in this PR.